harvester/molecularmatch_trials.py

```diff
@@ -140,8 +139,7 @@ def convert(evidence):
         genes = set([])
         for t in evidence_tags:
             if t['facet'] == 'GENE':
-                if gene_enricher.get_genes(t['term']):
-                    genes.add(t['term'])
+                genes.add(t['term'])
```

Comment:
I know that this is already merged, and in most cases this should be fine (genes are the most harmonized concept between resources without any additional effort). However, I feel like we should be using the normalized term provided by `get_genes` instead of just adding directly. If an error is thrown (due to term ambiguity), we should handle it here.

Reply:
This code was removed from the harvester to centralize the genes handling during normalization.